examples/add_person.d

```diff
@@ -9,7 +9,10 @@ import tutorial.addressbook;
 /// This function fills in a Person message based on user input.
 Person promptForAddress()
 {
-    auto person = new Person;
+    static if (is(Person == class))
```

Comment:
Three times the same `static if/else`, maybe a small template `make!T`.
Ex.: `auto person = make!Person`